templates/flutter/lib/client.dart.twig

```diff
@@ -128,7 +128,7 @@ class Client {
             throw {{spec.title | caseUcfirst}}Exception(e.message);
           }
           if(responseType == ResponseType.bytes) {
-            fif(e.response.headers['content-type'].contains('application/json')) {
+            if(e.response.headers['content-type'].contains('application/json')) {
               final res = json.decode(utf8.decode(e.response.data));
```

Comment:
Do we need to explicitly use `utf8.decode` on any JSON response?

Reply:
Yes, not any JSON response, but this, when we set ResponseType as bytes for Dio, the return value is utf8 encoded bytes instead of JSON string, (which we require for binary files) so even JSON response is encoded as bytes so we need to decode here

Reply:
OK